First set of edits to notebook

anacmontoya opened the pull request:
I added markdown cells to make code more readable. Also tried to follow the project pythia cookbook template

ThomasMGeo commented:
Hi @anacmontoya !

This looks great! 

We will need to add `fastparquet` to environment.yml (that is the check that is failing).

**Besides that, let's work together to add some more details in these areas:**

0. Maybe add some information on what the columns are? They are from numeric weather output. But be good to get a quick recap on what each one is. 
1. Add some more information and resources (maybe a link or two) on why validation and testing datasets are best practice, and not just a testing dataset.
2. Maybe add a link to the docs for standard scaler, and 1-2 sentences explaining what it is doing. 
3. Move the SVC import down to where we use it.

anacmontoya commented:
Second commit:

1. explained names of columns
2. added more info/resources on training/validation sets
3. explained StandardScaler function and why we scale data
4. moved SVC import
5. added references

Notes:
1. Should we change title and add a picture at the beginning of the notebook?
2. still missing pre-requisites section, any thoughts?

ThomasMGeo commented:
Hi @anacmontoya , I merged it! It looks much improved. 

Let's brainstorm your other comments, and we can do that in a new PR.
